pkgs/development/libraries/audec/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, libsndfile, libsamplerate, meson, ninja, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  pname = "audec";
+  version = "0.2";
+
+  src = fetchurl {
+    url =  "https://git.zrythm.org/cgit/libaudec/snapshot/libaudec-${version}.tar.gz";
```

Comment:
This sounds like generated tarball. `fetchgit` will likely be more reliable since it hashes the directory tree instead of the tarball.